pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/QueryRunnerTestBase.java

```diff
@@ -98,6 +98,10 @@ protected List<Object[]> queryRunner(String sql) {
   protected List<Object[]> queryH2(String sql)
       throws Exception {
     Statement h2statement = _h2Connection.createStatement(ResultSet.TYPE_FORWARD_ONLY, ResultSet.CONCUR_READ_ONLY);
+    // TODO: Hack. Let's discuss how best to support this.
+    if (sql.contains("option")) {
+      sql = sql.replaceAll("option\\(useColocatedJoin\\=true\\)", "");
```

Comment:
I added this to ensure basic sanity tests are passing. Proper tests for colocated join would be quite complex and I don't know if we have a ready to use framework.

I can undo this change and pick up the tests in a separate PR since they'll take time.

Reply:
^ Essentially the issue is that the H2 queries fail because of the option to enable useColocatedJoin.